desc/protoprint/print.go

```diff
@@ -1276,7 +1281,11 @@ func (p *Printer) printReservedRanges(ranges []reservedRange, maxVal int32, addr
 	_, _ = fmt.Fprintln(w, ";")
 }
 
-func (p *Printer) printReservedNames(names []string, addrs []elementAddr, w *writer, sourceInfo internal.SourceInfoMap, parentPath []int32, indent int) {
+func useQuotedReserved(fd *desc.FileDescriptor) bool {
+	return fd.AsFileDescriptorProto().GetEdition().Number() < descriptorpb.Edition_EDITION_2023.Number()
```

Comment:
nit: enum values can be directly compared
```suggestion
	return fd.AsFileDescriptorProto().GetEdition() < descriptorpb.Edition_EDITION_2023
```

Reply:
Whoops!